packages/actions/test/unit/authentication.test.ts

```diff
@@ -85,10 +90,63 @@ describe("Authentication", () => {
             )
         })
 
+        it("should not be possible to authenticate with an incorrect password", async () => {
```

Comment:
These four tests should also be moved in another describe block. Could be called: `signInWithEmailAndPassword` or anything else you see fit

Reply:
I removed that test cases in 33230aabd0794e6a077432b03368da7d478fcf71 because actually they are a testing Firebase's feature, not p0tion. Let me know if you think we need to test on the p0tion side by some reason.